src/components/NormalRightRow/index.jsx

```diff
@@ -21,7 +21,7 @@ function NormalRightRow({ schema, classes }) {
             component="div"
             variant="subtitle2"
             className={classes.line}>
-            <br />
+            {null}
           </Typography>
```

Comment:
Does the following work?

```diff
diff --git a/src/components/NormalLeftRow/index.jsx b/src/components/NormalLeftRow/index.jsx
index 1246094..085039b 100644
--- a/src/components/NormalLeftRow/index.jsx
+++ b/src/components/NormalLeftRow/index.jsx
@@ -29,13 +29,7 @@ function NormalLeftRow({ schema, classes }) {
   keywords.forEach((keyword, i) => {
     if (i > 0) {
       blankLinePaddings.push(
-        <Typography
-          key={`${keyword} line`}
-          component="div"
-          variant="subtitle2"
-          className={classes.line}>
-          {null}
-        </Typography>
+        <div key={`${keyword} line`} className={classes.line} />
       );
     }
   });
```

If it does, feel free to do the same thing for the other places.

Reply:
Yep! I've made sure that this works since each of the lines and rows have a minHeight property to match a single line or row 😄

Reply:
Ah, what I meant by "Does the following work?" was the suggested diff just below it that replaces the Typography component with a div element. I was thinking it's not necessary to use a Typography element when not rendering text.

Reply:
Oh, my misunderstanding! 
Checked that the div can replace the Typography component (made sure this works). Will be implementing such changes in PR #38 :)